examples/illustrations/device/device.json

```diff
@@ -25,21 +26,27 @@
                     "@type": "uco-observable:OperatingSystemFacet",
                     "uco-core:name": "Windows 7 Ultimate Edition",
                     "uco-observable:manufacturer": "Microsoft",
-                    "uco-observable:version": "6.1.7601 Service Pack 1 Build 7601"
+                    "uco-observable:version": "6.1.7601 Service Pack 1 Build 7601",
+                    "uco-observable:installDate":
+                    {
+                        "@type": "xsd:dateTime",
+                        "@value": "2019-07-10T16:33:42Z"
+}
                 },
                 {
                     "@type": "uco-observable:ComputerSpecificationFacet",
-                    "uco-observable:biosVersion": "E1762IMS.10M",
+                    "uco-observable:firmwareVersion": "E1762IMS.10M",
```

Comment:
I don't quite see why you changed this.  Searching the web for that string seems to indicate it is properly a bios version.  Do you have a reason you think it should be `firmwareVersion` instead?

Reply:
Ill fix this one now thanks for review